### PR TITLE
New e2e and unit test

### PR DESCRIPTION
**Pull request by mmadariaga**
qa: added some pending e2e test
qa: added video recording feature for a better error debugging

### CODE REVIEW

File: web/admin/application/configs/klear/model/HolidayDates.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ production:
           order:
             Calendar.name: asc
     name:
-      title: ngettext('Name', 'Names', 1)
+      title: _('Name', 'Names', 1)
```
Comment (Kaian) [inline web/admin/application/configs/klear/model/HolidayDates.yaml]: 🤔

File: library/Ivoz/Kam/Domain/Model/Rtpproxy/Rtpproxy.php.orig [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return $this->id;
+    }
+
+    public function setSetid($setid)
+    {
+        Assertion::regex($setid, '/^[0-9]{1,3}$/');
+        return parent::setSetid($setid);
+    }
+}
+
```
Comment (Kaian) [inline library/Ivoz/Kam/Domain/Model/Rtpproxy/Rtpproxy.php.orig]: This is a .orig file. Is this required?

File: library/Ivoz/Kam/Domain/Model/TrunksUacreg/TrunksUacreg.php.orig [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+     * @param string $authProxy
+     * @return TrunksUacregAbstract
+     */
+    public function setAuthProxy($authProxy)
+    {
+        Assertion::regex($authProxy, '/^sip:.+$|^sips:.+$/');
+        return parent::setAuthProxy($authProxy);
+    }
+}
+
```
Comment (Kaian) [inline library/Ivoz/Kam/Domain/Model/TrunksUacreg/TrunksUacreg.php.orig]: More .php.orig 🤔

**Comment by mmadariaga** [comment]
@Kaian It should be already fixed. Thanks for the review :)

File: library/Ivoz/Provider/Domain/Model/Company/Company.php [hunk 4]
```diff
@@ -21,21 +22,53 @@ class Company extends CompanyAbstract implements CompanyInterface
...
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDomainUsers()
+    {
+        $value = parent::getDomainUsers();
+        return trim($value);
+    }
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Model/Company/Company.php]: If we trim the setters it wouldn't be required to trim the getters. I think we should choose (I personally prefer setters tho).

Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Model/Company/Company.php]: I agree

**Pull request merged**